.env.example

```diff
@@ -0,0 +1,38 @@
+# Get key from https://platform.openai.com/account/api-keys
+# This gives the user access to the OpenAI API, which is necessary for the Agent to function. 
+OPENAI_API_KEY=<YOUR_API_KEY>
+
+# For a structure running in Griptape Cloud, set this variable to: https://cloud.griptape.ai
+# If running against Skatepark, set this environment variable to tell the client which URL Skatepark is listening on.
+# Default is https://127.0.0.1:5000, but is configurable when running 'gt skatepark start'. 
+GT_CLOUD_BASE_URL=
+
+# The Skatepark emulator does not require Griptape Cloud API Key.  
+# Structures in Griptape Cloud require a valid API key to authorize your client to call them. 
+# Generate a key from https://cloud.griptape.ai/account/api-keys 
+GT_CLOUD_API_KEY=<YOUR_API>
+
+# Structure ID from the Griptape Cloud or Skatepark
+# For Griptape Cloud: get from https://cloud.griptape.ai/structures 
+# For Skatepark: 
+# Once skatepark is started with 'gt skatepark start' 
+# get the id from running 'gt skatepark register --main-file <path-to-main-file>'
+GT_STRUCTURE_ID=
+
+# When running in Griptape Cloud, the Structure first reports
+# a status of QUEUED before transitioning to RUNNING. Skatepark
+# emulates this behavior by applying a default time to remain 
+# in the QUEUED state in order to allow your client to 
+# handle this state appropriately. By default, Skatepark
+# keeps your Structure QUEUED for 2 seconds.
+# You can override the time that the Structure remains in
+# the QUEUED state within Skatepark by setting this environment
+# variable to the desired time in seconds.
+GT_SKATEPARK_QUEUE_DELAY=2
+
+# Allows the user to retrieve the session ID from the CDK in order to interact with the DynamoDB table
```

Comment:
it's not coming from the CDK directly, the CDK creates the resource for us and then they need to get it, correct?

Reply:
correct - fixing now